R/project_visualiser.R

```diff
@@ -1018,8 +1021,18 @@ define_app_server <- function(network_object, project_path) {
           function_name <- input$aiAssist
           tab_name <- paste(function_name)
 
+          # Create a new environment to avoid sourcing scripts into the namespace
+          pkg_env <- new.env(parent = baseenv())
+
+          print("-----------------")
+          # get the full function paths including project path and use from here on wards
+          foo_path <- paste0(project_path, "/", foo_path)
```

Comment:
The function `file.path()` is faster and platform-independent.

`file.path(project_path, foo_path)`

Reply:
It seems that this line is no longer relevant!

Reply:
Yeah I think these changed - maybe it was an old commit?

Reply:
Yes, it was from an old commit.
I started reviewing by looking at a few old commits to track the logic behind the fixes.